Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1077,98 +1071,205 @@ private protected void UpdateMaterialAssetByDefault(MaterialData src, ModelImpor
+            }
+            else if (propNameID == MeshSyncConstants._BumpMap) {
+                HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._NORMALMAP);
+            }
+            else if (propNameID == MeshSyncConstants._ParallaxMap) {
+                HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._PARALLAXMAP);
+            }
+#if AT_USE_HDRP
+            else if (propNameID == MeshSyncConstants._EmissiveColorMap) {
+                Color baseEmissionColor = Color.white;
+
+                if (materialProperties.TryGetValue(MeshSyncConstants._EmissionColor, out var emissionColorProp)) {
+                    baseEmissionColor = emissionColorProp.vectorValue;
+                }
+
+                float emissionStrength = 0;
+                if (materialProperties.TryGetValue(MeshSyncConstants._EmissionStrength, out var emissionStrengthProp)) {
+                    emissionStrength = emissionStrengthProp.floatValue;
+                }
+
+                destMat.SetFloat(MeshSyncConstants._UseEmissiveIntensity, 1);
+                destMat.SetFloat(MeshSyncConstants._EmissiveIntensityUnit, 0);
+                destMat.SetFloat(MeshSyncConstants._EmissiveIntensity, emissionStrength);
+                destMat.SetColor(MeshSyncConstants._EmissiveColorLDR, baseEmissionColor);
+                destMat.SetColor(MeshSyncConstants._EmissiveColor, baseEmissionColor * emissionStrength);
+            }
+            else if (propNameID == MeshSyncConstants._HeightMap) {
+                if (prop.type == MaterialPropertyData.Type.Texture) {
+                    MaterialPropertyData.TextureRecord rec = prop.textureValue;
+                    Texture2D                          tex = FindTexture(rec.id, textureHolders);
+                    if (tex != null) {
+                        destMat.EnableKeyword(MeshSyncConstants._HEIGHTMAP);
+
+                        // If there is no displacement mode set, set it to vertex displacement:
+                        if (!destMat.IsKeywordEnabled(MeshSyncConstants._PIXEL_DISPLACEMENT)) {
+                            destMat.EnableKeyword(MeshSyncConstants._VERTEX_DISPLACEMENT);
+                            destMat.SetInt(MeshSyncConstants._DisplacementMode, 1);
                         }
-                        break;
-                    default: break;
+
+                        float scale = 10;
+                        if (materialProperties.TryGetValue(MeshSyncConstants._Parallax, out var heightScaleProp)) {
+                            scale = heightScaleProp.floatValue;
+                        }
+
+                        destMat.SetFloat(MeshSyncConstants._HeightMin, -scale);
+                        destMat.SetFloat(MeshSyncConstants._HeightMax, scale);
+
+                        float amplitude = scale * 2;
+                        destMat.SetFloat(MeshSyncConstants._HeightAmplitude, amplitude * 0.01f);
```

Comment:
If this code is new, can you write in a comment why we are multiplying with 2, and then with 0.01f ?
For example, if that's a unit conversion ?

Reply:
Added comments